networkx/algorithms/triads.py

```diff
@@ -394,6 +492,14 @@ def random_triad(G):
     -------
     G2 : subgraph
        A randomly selected triad (order-3 NetworkX DiGraph)
+
+    Examples
+    --------
+    >>> G = nx.DiGraph([(1, 2), (1, 3), (2, 3), (3, 1), (5, 6), (5, 4), (6, 7)])
+    >>> Random = nx.random_triad(G)
+    >>> nx.edges(Random)
```

Comment:
```suggestion
    >>> triad = nx.random_triad(G)
    >>> nx.edges(triad)
```

explicit name for the variable.